faq.md

```diff
@@ -0,0 +1,31 @@
+## FAQ questions
+
+### Why use a new ABI? Why not an attribute?
+- Unlike ABIs, there is no syntax for assigning attributes to function
+  pointers. The ABI must be part of the type system so that callers of function
+  pointers know whether or not the function may unwind.
+### What should I do if I want to have C++ functions that unwind into Rust? [DRAFT; TODO]
+- mark declarations in Rust `extern "C unwind"`
+- Do *not* use `catch_unwind` in calling code
+- compile w/ `panic = unwind`
+- Check compiler, linker, & platform documentation (TODO: expand on this)
+- Re-test for every compiler update
+- (TODO: other caveats?)
+### How does cross-language unwinding differ from cross-language
+  `setjmp`/`longjmp`?
+- `setjmp`/`longjmp` across Rust frames is currently guaranteed not to have
+  undefined behavior or to `abort`
```

Comment:
This is a weird way to say that the behavior of `setjmp`/`longjmp` is defined. Maybe just explain how their behavior is defined instead?

Reply:
"Not undefined" is not equivalent to "is defined"; this is an (arguably unfortunate) legacy of the C/C++ ISO standards. Nico and I agree that Rust needs to better specify how "unspecified", "undefined", "implementation defined", etc are to be used in the Rust Reference and elsewhere in the Rust community, but at the moment, I felt that this was the safest phrasing, because "undefined behavior" is (ironically) the best-defined phrase at hand, as far as I can tell.

Reply:
Sure, but what I'm saying is that the behavior of `setjmp` and `longjmp` in Rust is actually defined.